app/src/App.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Please remove `useEffect` if not using it anywhere.

Reply:
Why though?  I'm curious what it matters?  It is likely being imported so that it could have been used eventually.

Reply:
To me personally it is a trivial good practice in order to make code leaner, more readable, and more maintainable. It's a good idea to work at building the habit of stripping away everything unused. 

Here's an interesting discussion that's relevant here (didn't know that `import { something } from "library" imports and bundles the entire library`):

https://stackoverflow.com/questions/34239731/how-to-minimize-the-size-of-webpacks-bundle/34241128#34241128